File: tests/fixtures/6588/solution.py
class Solution:
    _MAX = 1_000_001
    _sieve: list = []

    @classmethod
    def _build_sieve(cls) -> list:
        sieve = bytearray(cls._MAX)
        sieve[0] = sieve[1] = 1
        for i in range(2, cls._MAX):
            if not sieve[i]:
                for j in range(i * i, cls._MAX, i):
                    sieve[j] = 1
        return sieve

    def solve(self, n: int) -> str:
        if not Solution._sieve:
            Solution._sieve = self._build_sieve()
        sieve = Solution._sieve
        a = 3
        while a <= n // 2:
            if not sieve[a] and not sieve[n - a]:
                return f"{n} = {a} + {n - a}"
            a += 2
        return "Goldbach's conjecture is wrong."

File: tests/fixtures/6588/Solution.java
public class Solution {
    private static final int MAX = 1_000_001;
    private static final boolean[] notPrime = new boolean[MAX];

    static {
        notPrime[0] = notPrime[1] = true;
        for (int i = 2; i < MAX; i++) {
            if (!notPrime[i]) {
                for (long j = (long) i * i; j < MAX; j += i) {
                    notPrime[(int) j] = true;
                }
            }
        }
    }

    public String solve(int n) {
        for (int a = 3; a <= n / 2; a += 2) {
            if (!notPrime[a] && !notPrime[n - a]) {
                return n + " = " + a + " + " + (n - a);
            }
        }
        return "Goldbach's conjecture is wrong.";
    }
}
